h2o-hadoop-3/assemblyjar.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
     api project(":h2o-ext-steam")
     api "org.apache.hadoop:hadoop-client:$hadoopMavenArtifactVersion"
     api "org.apache.hadoop:hadoop-mapreduce-client-app:$hadoopMavenArtifactVersion"
+    api "org.apache.hadoop:hadoop-aws:$hadoopMavenArtifactVersion"
```

Comment:
Isn't `aws-java-sdk` also needed if we have `hadoop-aws` as dependency?

Reply:
these are already provided by project h2o-persist-s3 (S3 a such was working even before), what was missing was S3AFileSystem which is provided by this dependency